templates/thumbor.conf.j2

```diff
@@ -469,6 +469,9 @@ UPLOAD_PUT_ALLOWED = {{ thumbor_upload_put_allowed }}
 ################################################################################
 
 ############################### Thumbor AWS ####################################
+{% if s3_aws_region is defined %}
```

Comment:
What do you think of adding the `thumbor_` prefix to the variable name?

Reply:
I think that'll be nice to have. I'll update the variable